junos/data_source_system_information.go

```diff
@@ -0,0 +1,85 @@
+package junos
+
+import (
+	"context"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func dataSourceSystemInformation() *schema.Resource {
+	return &schema.Resource{
+		ReadContext: dataSourceSystemInformationRead,
+
+		Schema: map[string]*schema.Schema{
+			"hardware_model": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"os_name": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"os_version": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"serial_number": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"host_name": {
```

Comment:
`host_name` is not used. Hostname is filled in ID, can you remove this argument ?

Reply:
Thanks for catching this, i totally missed it. Thanks for the input, I have removed the attribute.